### ISSUE TITLE
IOS black screen

### ISSUE THREAD

**Issue by leobispo** [Furkankyl/twitter_qr_scanner#3]
Hi,

I am trying to run the app example on the IOS but I am getting a black screen. No camera is getting started.

Any ideas?

**Comment by leobispo** [comment]
Posted a PR for you.

If you can't merge it, please let me know so I will fork the project

**Comment by Streissi** [comment]
@leobispo i know this is old, but did you forked the project and created a new pub package? I need a good qrcode scanner plugin for android and ios but can not find anyone that works good.

**Comment by leobispo** [comment]
@Streissi No, I didn't fork (I embedded it on my project), but you can get this PR and use this plugin. It is working :). If you need help just let me know
